opentelemetry-sdk/src/opentelemetry/sdk/logs/__init__.py

```diff
@@ -112,6 +115,133 @@ def force_flush(self, timeout_millis: int = 30000):
         """
 
 
+class SynchronousMultiLogProcessor(LogProcessor):
+    """Implementation of class:`LogProcessor` that forwards all received
+    events to a list of log processors sequentially.
```

Comment:
Should this class be named `SequentialMultiLogProcessor`?

Reply:
I think the current name is more accurate as we are synchronously calling one log processor after other.